repose-aggregator/components/filters/add-header/src/main/resources/META-INF/schema/config/add-header.xsd

```diff
@@ -30,8 +30,8 @@
 
     <xs:complexType name="AddHeadersConfig">
         <xs:all>
-            <xs:element name="request" type="HttpMessage"/>
-            <xs:element name="response" type="HttpMessage"/>
+            <xs:element name="request" type="HttpMessage" minOccurs="0"/>
+            <xs:element name="response" type="HttpMessage" minOccurs="0"/>
```

Comment:
This should probably have the logic to ensure that there is at least one of them.